gh-weblog/components/Editor.jsx

```diff
@@ -12,25 +13,30 @@ export default createClass({
   render() {
     return (
       <textarea
-        ref="textarea"
+        ref={this.refs.textarea}
         className="editor"
         hidden={this.props.hidden}
         value={this.state.text}
-        onChange={this.record}
+        onInput={this.record}
       />
     );
   },
 
   setText(text) {
-    const textarea = this.refs.textarea.getDOMNode();
+    const textarea = this.refs.textarea.current;
```

Comment:
I wasn't a fan when React changed this either: "current _what_?". With getDOMNode(), you knew what you were asking for. With just `this.refs.textarea` it would make sense because that's the reference so that should reasonably be a DOM node (because we already have the component: it's literally just "this"). This was a bad API change on React's part.

Reply:
It's a consequence of refs now being objects of their own (`createRef` returns a `Ref`). I guess they kinda function as "boxes" now?

The `current` property isn't just a straight accessor for `getDOMNode`; it's the value inside the ref. If you have a `ref` on a component, that'll also have a `current` property.